src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const InputWithLabel = (props) => {
+  const inputRef = React.useRef();
+
+  React.useEffect(() => {
+    inputRef.current.focus();
+  });
```

Comment:
There's nothing wrong with this, but another method of doing this is to import `useRef` and `useEffect`

```suggestion
import React, { useRef, useEffect } from 'react';

const InputWithLabel = (props) => {
  const inputRef = useRef();

  useEffect(() => {
    inputRef.current.focus();
  });
```